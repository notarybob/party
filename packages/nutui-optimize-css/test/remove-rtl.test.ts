import postcss from 'postcss'
import { describe, expect, it } from 'vitest'
import optimizeCss from '../dist/index.cjs'

const css = `
[dir=rtl] .ca, .xcdd {
  margin-left: 0;
  margin-right: 9px
}
[dir=rtl] .nut-address-exist-item-info, .nut-rtl .nut-address-exist-item-info {
  margin-left: 0;
  margin-right: 9px
}
`
describe('@nutui/optimize-css', () => {
  it('remove rtl', async () => {
    const a = await postcss([
      optimizeCss({
        removeRtl: true,
      }),
    ]).process(css, { from: undefined })
    const optimizedCsss = a.css.toString()
    // @ts-ignore
    expect(optimizedCsss).toMatchSnapshot()
  })
})
