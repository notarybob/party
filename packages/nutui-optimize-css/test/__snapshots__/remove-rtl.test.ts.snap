// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`@nutui/optimize-css > remove rtl 1`] = `
"
"
`;
